maven/maven_manager.go

```diff
@@ -121,35 +121,41 @@ func NewWrapperMavenManager(appPath string, logger bard.Logger) WrapperMavenMana
 }
 
 // ShouldInstall the Maven Wrapper
-func (s WrapperMavenManager) ShouldInstall() bool {
-	command := filepath.Join(s.appPath, "mvnw")
+func (w WrapperMavenManager) ShouldInstall() bool {
+	command := filepath.Join(w.appPath, "mvnw")
 	_, err := os.Stat(command)
 	return err == nil
 }
 
 // Install the Maven wrapper tool
 // Slightly misleading as this doesn't install anything, it just makes sure the wrapper can be run
 // The wrapper itself handles any installation, if it's necessary
-func (s WrapperMavenManager) Install() (string, libcnb.LayerContributor, *libcnb.BOMEntry, error) {
-	command := filepath.Join(s.appPath, "mvnw")
+func (w WrapperMavenManager) Install() (string, libcnb.LayerContributor, *libcnb.BOMEntry, error) {
+	command := filepath.Join(w.appPath, "mvnw")
 
 	if err := os.Chmod(command, 0755); err != nil {
-		s.logger.Bodyf("WARNING: unable to chmod %s:\n%s", command, err)
+		w.logger.Bodyf("WARNING: unable to chmod %s:\n%s", command, err)
 	}
 
-	if err := s.cleanMvnWrapper(command); err != nil {
-		s.logger.Bodyf("WARNING: unable to clean mvnw file: %s\n%s", command, err)
+	if err := w.cleanMvnWrapper(command); err != nil {
+		w.logger.Bodyf("WARNING: unable to clean mvnw file: %s\n%s", command, err)
+	}
+
+	wrapperProperties := filepath.Join(w.appPath, ".mvn/wrapper/maven-wrapper.properties")
+	if _, err := os.Stat(wrapperProperties); err == nil {
+		if err := w.cleanMvnWrapper(wrapperProperties); err != nil {
+                	w.logger.Bodyf("WARNING: unable to clean wrapper properties file: %s\n%s", wrapperProperties, err)
+        	}
```

Comment:
Fantastic catch. Can we get a unit test to cover this?

Reply:
I looked at this but didn't add one as the current unit test is testing the cleanMvnWrapper func. This change is simply calling that a second time. But I can add one if you think there's a gap still